b_cfn_elasticsearch_cloner/cloner_source/index.py

```diff
@@ -13,34 +14,55 @@
 try:
     from dynamodb_decode import DynamodbDecoder
 except ImportError as ex:
-    logger.exception("Failed import.")
+    logger.exception('Failed import.')
     from b_cfn_elasticsearch_cloner.cloner_source.dynamodb_decode import DynamodbDecoder
 
+ES_INDEX = os.environ['ES_INDEX_NAME']
+ES_ENDPOINT = os.environ['ES_DOMAIN_ENDPOINT']
+PRIMARY_KEY_FIELD = os.environ['PRIMARY_KEY_FIELD']
+REGION = os.environ['AWS_REGION']
 
-es_index = os.environ["ES_INDEX_NAME"]
-es_endpoint = os.environ["ES_DOMAIN_ENDPOINT"]
-primary_key_field = os.environ["PRIMARY_KEY_FIELD"]
-region = os.environ["AWS_REGION"]
+SAGEMAKER_ENDPOINT_NAME = os.environ.get('SAGEMAKER_ENDPOINT_NAME')
+SAGEMAKER_EMBEDDINGS_KEY = os.environ.get('SAGEMAKER_EMBEDDINGS_KEY')
+EMBEDDINGS_KEY = os.environ.get('EMBEDDINGS_KEY')
 
+sagemaker_client = boto3.client('sagemaker-runtime')
 
 credentials = boto3.Session().get_credentials()
```

Comment:
Put `boto3.Session()` to a separate variable and use it to initialize `sagemaker-runtime` and get credentials. After all, they both should share the same boto session.